node/src/Transaction.ts

```diff
@@ -347,6 +348,20 @@ export class BaseTransaction<T extends BaseTransaction<T>> {
         return this.addAndReturn(createMSet(keyValueMap));
     }
 
+    /**
+     * Sets multiple keys to values if the key does not exist. The operation is atomic, and if one or
+     * more keys already exist, the entire operation fails.
+     *
+     * See https://valkey.io/commands/msetnx/ for more details.
+     *
+     * @remarks When in cluster mode, all keys in `keyValueMap` must map to the same hash slot.
```

Comment:
We don't need this warning for the transaction implementation, just the BaseClient one
```suggestion
```